src/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital,wght@0,400;0,700;1,400&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital,wght@0,400;0,700;1,400&family=Mr+De+Haviland&family=Roboto&display=swap" rel="stylesheet">
+    <title>Chrimpo</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="logo"> <a href="index.html">chrimpo</a></div>           
+                <ul class="nav-list">
+                    <li><a id="seafood-meals" href="index.html"> Seafood meals</a></li>
+                    <li><a id="chicken-meals" href="#"> chicken meals</a></li>
+                    <li><a id="Meat-meals" href="#">Meat meals</a></li>
+                </ul>
+            </a>
+        </nav>
+        </header>
+
+    <main>
+        <section id="popup">
+        </section>
+        <section class=home-page>
+            <ul class="meals">
+            </ul>
+        </section>
+    </main>
+    <footer>
+        <h3>
+            <span>Created By</span>
+            <a href="https://github.com/farouqdaif1">Farouq Daif</a>
+            &amp;
```

Comment:
Please follow the mock-up for the Home page footer. It should be on the left side of the page.

Reply:
Changes Done. Thank you.

Reply:
Thank you